# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-contain/crashtests/contain-dynamic-change-crash.html
# Commit: 421d7bf24007
# Full Hash: 421d7bf240078ff30871183289ce376441fadb09
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-11-14 21:16:19
# Description:
#   Bug 1928724 - Ensure we always have a float manager during block-frame reflow. r=layout-reviewers,emilio
#   
#   Normally we shouldn't be starting reflow from a block that is not a BFC,
#   and so BlockNeedsFloatManager would return true for the reflow root.
#   
# ==============================================================================

diff -r 19f297419354 -r 421d7bf24007 testing/web-platform/tests/css/css-contain/crashtests/contain-dynamic-change-crash.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-contain/crashtests/contain-dynamic-change-crash.html	Thu Nov 14 10:44:48 2024 +0000
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="help" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1928724">
+
+<style>
+*:only-child {
+  contain: size layout paint;
+}
+</style>
+
+<script>
+document.addEventListener("DOMContentLoaded", () => {
+  a.getBoundingClientRect();
+  a.appendChild(c);
+  b.scrollBy();
+})
+</script>
+
+<body>
+<svg>
+<animateMotion id="b">
+</animateMotion>
+<foreignObject id="a">
+<article>
+<data id="c">
